modules/convert/notification.go

```diff
@@ -47,6 +47,13 @@ func ToNotificationThread(n *models.Notification) *api.NotificationThread {
 			if err == nil && comment != nil {
 				result.Subject.LatestCommentURL = comment.APIURL()
 			}
+
+			pr, _ := n.Issue.GetPullRequest()
```

Comment:
Is there a pull.State() or should we create one?

Reply:
there is a pull.State:
```go
	PullRequestStatusConflict PullRequestStatus = iota
	PullRequestStatusChecking
	PullRequestStatusMergeable
	PullRequestStatusManuallyMerged
	PullRequestStatusError
	PullRequestStatusEmpty
```
but this is not that helpfull info, if you like to build a UI via API showing notifications of issues/pulls